### PR TITLE
LPS-82171 Add Unit Tests

### PR DESCRIPTION
**Pull request by pedroqueiroz**

### CODE REVIEW

**Comment by pedroqueiroz** [comment]
ci:test:sf

**Comment by pedroqueiroz** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-lists/dynamic-data-lists-test/src/test/java/com/liferay/dynamic/data/lists/internal/upgrade/v1_2_0/UpgradeDDLRecordSetTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+/**
+ * @author Pedro Queiroz
+ */
```
Comment (rafaprax) [inline modules/apps/dynamic-data-lists/dynamic-data-lists-test/src/test/java/com/liferay/dynamic/data/lists/internal/upgrade/v1_2_0/UpgradeDDLRecordSetTest.java]: Hey @pedroqueiroz, Is it really necessary to use PowerMock here? I think, we can call the `getNewActionIds ` method directly. What do you think? Thx

**Comment by rafaprax** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by rafaprax** [comment]
LGTM

**Comment by pedroqueiroz** [comment]
Sent to BChan at https://github.com/brianchandotcom/liferay-portal/pull/60153.

**Pull request closed without merging**